src/index.html

```diff
@@ -5,8 +5,332 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          class="header__logo"
+          alt="Moyo logo"
+        >
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a href="#apple" class="header__nav-link is-active">Apple</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="#samsung"
+            class="header__nav-link"
+            >Samsung
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="#smartphones"
+            class="header__nav-link"
+            >Smartphones
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#laptops&computers"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >Laptops & computers
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="#gadgets" class="header__nav-link">Gadgets</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="#tablets" class="header__nav-link">Tablets</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="#photo" class="header__nav-link">Photo</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="#video" class="header__nav-link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <div class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+          class="card__img"
+        >
+        <h2 class="card__title">
```

Comment:
```suggestion
        >
        
        <h2 class="card__title">
```

Reply:
Fix this everywhere
<img width="1065" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/86789161/6f6b9191-fc17-4b84-b655-50761c477833">